tests/Fixture/TestKernel.php

```diff
@@ -35,6 +35,13 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
     {
         parent::configureContainer($c, $loader);
 
+        $path = \sprintf('%s/config/%s.yaml', __DIR__, $this->getEnvironment());
+        if (file_exists($path)) {
+            $loader->load($path);
+        }
```

Comment:
```suggestion
        if ($this->getEnvironment() !== 'test') {
            $loader->load(\sprintf('%s/config/%s.yaml', __DIR__, $this->getEnvironment()));
        }
```
this way we will throw if the file does not exist

Reply:
the env isn't test for some reason, it's dev not sure if that's intended

Reply:
Not intended, maybe you can set it in the constructor, to be sure of what we need to exclude in the if